integrations/db/ojdbc/pom.xml

```diff
@@ -25,7 +25,7 @@
         <version>2.3.2-SNAPSHOT</version>
     </parent>
 
-    <artifactId>ojdbc</artifactId>
+    <artifactId>helidon-integrations-db-ojdbc</artifactId>
```

Comment:
This looks like an incompatible change  (I also see you had to change the examples).  Is the artifact rename something that should wait until 3.0?

Reply:
Romain asked for those renames, because they do not follow naming conventions.